rosdep/python.yaml

```diff
@@ -5809,6 +5817,19 @@ python3-tornado:
       packages: [tornado]
   rhel: ['python%{python3_pkgversion}-tornado']
   ubuntu: [python3-tornado]
+python3-tqdm:
+  alpine: [py3-tqdm]
+  debian:
+    stretch: [python3-tqdm]
+  fedora: [python3-tqdm]
+  osx:
+    pip:
+      packages: [tqdm]
+  ubuntu:
+    bionic: [python3-tqdm]
+    eoan: [python3-tqdm]
+    focal: [python3-tqdm]
+    groovy: [python3-tqdm]
```

Comment:
Instead of calling them out this way, I'll suggest we instead do:

```
ubuntu:
  '*': python3-tqdm
  xenial: null
```

That way we'll have it for new distros automatically.